src/Amazon.ECS.Tools/ECSUtilities.cs

```diff
@@ -379,7 +379,18 @@ public static async System.Threading.Tasks.Task EnsureClusterExistsAsync(IToolLo
                     if (data != null)
                     {
                         containerDefinition.LogConfiguration = new LogConfiguration();
-                        containerDefinition.LogConfiguration.LogDriver = data["containerPath"] != null ? data["containerPath"].ToString() : null;
+                        containerDefinition.LogConfiguration.LogDriver = null;
+
+                        // Added support for logDriver JSON key as fix for legacy bug where JSON key containerPath was used. 
+                        if (data["logDriver"] != null)
+                        {
+                            containerDefinition.LogConfiguration.LogDriver = data["logDriver"].ToString();
+                        }
+                        else if (data["containerPath"] != null)
```

Comment:
Add a comment about how we have this `else if` clause due to a legacy issue and removing it would cause a breaking change.

Reply:
Added the comment. Please review.